src/BigInt.chpl

```diff
@@ -0,0 +1,88 @@
+module BigInt {
```

Comment:
Don't really think it matters, but should we name this `BigIntMsg.chpl`? I am just thinking that we do that with most of our files handling message parsing.

Reply:
Sure, I can make that change